webdavd.c

```diff
@@ -1209,7 +1209,7 @@ static Response * createFdResponse(int fd, uint64_t offset, uint64_t size, const
 	char dateBuf[100];
 	getWebDate(date, dateBuf, 100);
 	addHeader(response, "Content-Type", mimeType);
-	addHeader(response, "DAV", "1");
+	addHeader(response, "DAV", "1,2");
```

Comment:
I thought of that, but I had hard time coming up with a situation where a client would not:
* Avoid using locking, which even if webdavd's locks are not perfect, not using them is still more likely to result in data loss than using them.
* Use the locks anyway, which is not made worse by level 2 support.

It would be nice to at least have level 2 support be an option.  I'd much rather be able to support MacOS as a client than not, and don't want to give that up because of a hypothetical possibility of incorrect level 2 support causing a problem.